client/src/javascript/components/modals/feeds-modal/DownloadRulesTab.js

```diff
@@ -149,79 +196,175 @@ class DownloadRulesTab extends React.Component {
     );
   }
 
-  getRuleFields() {
-    const errors = Object.keys(this.state.errors).map((errorID, index) => {
-      return (
-        <FormRow key={index}>
-          <FormError>{this.state.errors[errorID]}</FormError>
-        </FormRow>
+  getModifyRuleForm(rule) {
+    return (
+      <li className="interactive-list__item interactive-list__item--stacked-content feed-list__feed" key={rule._id}>
+        <FormRowGroup>
+          <FormRow>
+            <Textbox
+              id="label"
+              label={this.props.intl.formatMessage({
+                id: 'feeds.label',
+                defaultMessage: 'Label',
+              })}
+              defaultValue={rule.label}
+            />
+            <Select
+              disabled={!this.state.feeds.length}
+              id="feedID"
+              label={this.props.intl.formatMessage({
+                id: 'feeds.applicable.feed',
+                defaultMessage: 'Applicable Feed',
+              })}
+              defaultID={rule.feedID}>
+              {this.getAvailableFeedsOptions()}
+            </Select>
+          </FormRow>
+          <FormRow>
+            <Textbox
+              id="match"
+              label={this.props.intl.formatMessage({
+                id: 'feeds.match.pattern',
+                defaultMessage: 'Match Pattern',
+              })}
+              placeholder={this.props.intl.formatMessage(MESSAGES.regEx)}
+              defaultValue={rule.match}
+            />
+            <Textbox
+              id="exclude"
+              label={this.props.intl.formatMessage({
+                id: 'feeds.exclude.pattern',
+                defaultMessage: 'Exclude Pattern',
+              })}
+              placeholder={this.props.intl.formatMessage(MESSAGES.regEx)}
+              defaultValue={rule.exclude}
+            />
+            <Textbox
+              style={{backgroundColor: '#222222'}}
```

Comment:
No inline styles

Reply:
I have now removed the inline styling.

I think this was left over from the previous iteration and I dont think it was doing anything.